front/app/components/admin/InviteUsersWithSeatsModal/messages.ts

```diff
@@ -0,0 +1,46 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  giveCollaboratorRights: {
+    id: 'app.containers.admin.inviteUsersWithSeatsModal.giveCollaboratorRights',
+    defaultMessage: 'Give collaborator rights',
+  },
+  giveAdminRights: {
+    id: 'app.containers.admin.inviteUsersWithSeatsModal.giveAdminRights',
+    defaultMessage: 'Give admin rights',
+  },
+  descriptionMessage: {
+    id: 'app.containers.admin.inviteUsersWithSeatsModal.descriptionMessage',
+    defaultMessage:
+      'You are inviting {noOfUsers} {noOfUsers, plural, one {user} other {users}} with {seatType} rights. Based on how many users are included in the invitation, you may need to buy additional seats.',
```

Comment:
Not really, the first one is the number and the second one is to determine the plural / singular of the word user

Reply:
Aah, right. You could shorten it:

      'You are inviting {noOfUsers, plural, one {1 user} other {# users}} with {seatType} rights. Based on how many users are included in the invitation, you may need to buy additional seats.',

This is also useful in case you have a language where the number would be after 'users'. In that case the translator can move the number part accordingly.